scripts/cephscript/ceph_script.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+set +e
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+
+CEPH_DIR="/root/s3-tests"
+CEPH_INIPATH="$CEPH_DIR/seagates3conf.ini"
+CEPH_TEST_DATA="/root/cephtestfilev2.txt"
```

Comment:
we must have same set of tests for v2 and v4 rite....it's just difference signature calculation?

Reply:
We have more awsv4 testcases as comared to awsv2 ,hence the diffrence in testcase  number would be there.